build/UIKit/dll/UIKit.def

```diff
@@ -1164,6 +1164,6 @@ LIBRARY UIKit
         UIRequestTransactionProcessing
         UIOrientationFromString
         UIApplicationDidChangeDisplayModeNofication
-        EbrApplicationActivated
-        EbrApplicationLaunched
+        OnApplicationActivated
+        OnApplicationLaunched
```

Comment:
nit: move this up into the "Application entry points (StarboardXAML)" section

And let us use UI* naming convention to be consistent